spec/features/account_page_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+describe "Account page" do
+  before do
+    # Sign in
+    @user = User.create!(username: 'Larry', email: 'user@example.com', password: 'password', digest_subscriber: 'true')
+    visit '/sign_in'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.